src/components/help/help.pw.tsx

```diff
@@ -0,0 +1,354 @@
+      await expect(helpComponent(page)).toHaveAttribute("href", hrefLink);
+    });
+  });
+
+  [true, false].forEach((boolVal) => {
+    test(`should check when isFocused prop is passed as ${boolVal}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <HelpComponentTest isFocused={boolVal}>{tooltipText}</HelpComponentTest>
+      );
+      const tooltip = getDataElementByValue(page, "tooltip");
+      if (boolVal === true) {
+        await expect(tooltip).toBeVisible();
+      } else {
+        await expect(tooltip).not.toBeVisible();
+      }
+    });
+  });
+
+  [-1, 0].forEach((tabIndex) => {
+    test(`should check tabIndex as ${tabIndex}`, async ({ mount, page }) => {
+      await mount(<HelpComponentTest tabIndex={tabIndex} />);
+
+      if (tabIndex === -1) {
+        await page.keyboard.press("Tab");
+        await expect(helpComponent(page)).not.toBeFocused();
+      } else {
+        await page.keyboard.press("Tab");
+        await expect(helpComponent(page)).toBeFocused();
+      }
+    });
+  });
+
+  colors.forEach(([names, color]) => {
+    test(`should check tooltipBgColor as ${names}`, async ({ mount, page }) => {
+      await mount(
+        <HelpComponentTest tooltipBgColor={color} isFocused>
+          {tooltipText}
+        </HelpComponentTest>
+      );
+      const tooltip = getDataElementByValue(page, "tooltip");
+      await expect(tooltip).toHaveCSS("background-color", color);
+    });
+  });
+
+  colors.forEach(([names, color]) => {
+    test(`should check tooltipFontColor as ${names}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <HelpComponentTest tooltipFontColor={color} isFocused>
+          {tooltipText}
+        </HelpComponentTest>
+      );
+      const tooltip = getDataElementByValue(page, "tooltip");
+      await expect(tooltip).toHaveCSS("color", color);
+    });
+  });
+
+  (["top", "bottom", "left", "right"] as HelpProps["position"][]).forEach(
+    (position) => {
+      test(`should render tooltipFlipOverrides as ${position}`, async ({
+        mount,
+        page,
+      }) => {
+        await page.setViewportSize({ width: 700, height: 120 });
+        await mount(
+          <Box ml="250px">
+            <Help tooltipFlipOverrides={[position]} isFocused>
+              {`This tooltip is positioned ${position}`}
+            </Help>
+          </Box>
+        );
+        await page.evaluate(() => window.scrollTo(50, 50));
+        const tooltip = getDataElementByValue(page, "tooltip");
+        await expect(tooltip).toHaveAttribute("data-placement", position);
+      });
+    }
+  );
+
+  testData.forEach((tooltipId) => {
+    test(`should check tooltipId as ${tooltipId}`, async ({ mount, page }) => {
+      await mount(
+        <HelpComponentTest tooltipId={tooltipId} isFocused>
+          {tooltipText}
+        </HelpComponentTest>
+      );
+      const tooltip = getDataElementByValue(page, "tooltip");
```

Comment:
**suggestion: (non-blocking)** as above, it would be better to add a space to separate the mount from the assertion to follow the same pattern as with the other test cases.